src/lib.rs

```diff
@@ -205,6 +205,28 @@ pub struct Buffer<'a, D, W> {
 }
 
 impl<D: HasDisplayHandle, W: HasWindowHandle> Buffer<'_, D, W> {
```

Comment:
I'm a bit unsure whether these should go on the buffer or the surface? Or does it make sense both places?

Reply:
I'd say we should provide both independently. These are intrinsically different when (if?) we provide/support a proper abstraction for hardware scaling: #177.